src/tests/ftest/launch.py

```diff
@@ -1163,6 +1163,26 @@ def rename_logs(avocado_logs_dir, test_file):
             "Error renaming {} to {}: {}".format(
                 test_logs_dir, new_test_logs_dir, error))
 
+    xml_file = os.path.join(new_test_logs_dir, "results.xml")
+    try:
+        with open(xml_file) as xml_buffer:
+            xml_data = xml_buffer.read()
+    except OSError as error:
+        print(
+            "Error opening {} : {}".format(xml_file, str(error)))
+        return
+
+    test_dir = os.path.split(os.path.dirname(test_file))[-1]
+    org_class = "classname=\""
+    new_class = "classname=\"FTEST_{}.".format(test_dir)
```

Comment:
Only if you have to rebase.
Minor: You can use `org_class` inside `new_class = "classname=\"FTEST_{}.".format(test_dir)` as well, as it is already declared before.

Reply:
sure.